sympy/physics/continuum_mechanics/beam.py

```diff
@@ -1521,6 +1528,91 @@ def plot_loading_results(self, subs=None):
         return PlotGrid(4, 1, ax1, ax2, ax3, ax4)
 
 
+    def draw(self):
+        if matplotlib is None:
+            raise ImportError('Install matplotlib to use this method.')
+
+        if numpy is None:
+            raise ImportError('Install numpy to use this method.')
+
+
+        length = self.length
+        height = 0.08
+
+        fig = matplotlib.pyplot.figure()
+        ax = fig.add_subplot(111, xbound=(0, 50), ybound=(0,1))
+        rect = matplotlib.patches.Rectangle((0, 0), length, height, facecolor="brown")
+        ax.add_patch(rect)
+
+        self._draw_load_arrows(ax)
+        self._draw_supports(ax)
+        matplotlib.pyplot.axis("off") # to turn off axis
+        ax.set_ylim(-0.5, 0.5)
+        matplotlib.pyplot.show()
```

Comment:
This method should return ax (standard practice in most packages that use matplotlib, e.g. pandas).

Reply:
Sure will make it return axis.